activiti-services/activiti-services-rest/src/main/java/org/activiti/services/rest/controllers/ProcessDefinitionController.java

```diff
@@ -73,4 +95,23 @@ public ProcessDefinitionResource getProcessDefinition(@PathVariable String id) {
         }
         return resourceAssembler.toResource(processDefinitionConverter.from(processDefinition));
     }
+
+    @RequestMapping(value = "/{id}/xml",
+            method = RequestMethod.GET)
+    public ProcessModelResource getProcessModel(@PathVariable String id) {
+        try (final InputStream resourceStream = repositoryService.getProcessModel(id)) {
+            String xml = new String(IoUtil.readInputStream(resourceStream, null), StandardCharsets.UTF_8);
```

Comment:
@daisuke-yoshimoto The endpoint should return the XML string, at least that is what I would expect as a client of this service. We don't need to return a resource. @erdemedeiros what do you think about that?